client/src/components/CommentApp/utils/layout.ts

```diff
@@ -51,9 +51,19 @@ export class LayoutController {
       return;
     }
 
+    // If the annotation's desired position node is not visible,
```

Comment:
Wondering if we should hide the comments if their fields are hidden?

Otherwise, would it be easy to expand any collapsed panels when the comment is focused?

Reply:
I'd be inclined to leave the comments shown - it doesn't look bad, and comments are the kind of thing we want to be easily found. If you don't want to see comments, then you can always leave comment mode!

Easyish, but I think we'd probably want to do it as a new subscriber for a) streamfield block and b) multifieldpanel listening for the focused comment being in the list of comments with contentpath starting with their contentpath. I don't think it would be too hard, I can think how to do it, but it is extra code that interacts with eg telepath - wondering if this is a step too far for the release candidate level of tweaks?

Reply:
Hmm, though with the first, I suppose for big streamfields with lots of comments it could be useful...